services/ingestion/etl/types/Project.js

```diff
@@ -2,19 +2,29 @@
 
 // Project model is discussed in EUBFR-4 EUBFR-5 and EUBFR-70
 
+type Coordinates<Geolocation> = Array<Geolocation | Coordinates<Geolocation>>;
+
 type Result = {
   result: string,
 };
 
 type Geolocation = {
```

Comment:
`Geolocation` needs to be updated: `type Geolocation = [number, number] | [number, number, number];` 

My 2 cents: it would make more sense to call this type `Coordinates` rather than `Geolocation` 👼